xml/admin_operating_pools.xml

```diff
@@ -64,825 +64,821 @@
   </para>
-
   <para>
-   For other cases, you can manually associate a free-form application name
-   with a pool:
+   To create a replicated pool, execute:
   </para>
-
-<screen>&prompt.cephuser;ceph osd pool application enable <replaceable>pool_name</replaceable> <replaceable>application_name</replaceable></screen>
-
-  <tip>
-   <title>Default application names</title>
-   <para>
-    &cephfs; uses the application name <literal>cephfs</literal>, &rbd; uses
-    <literal>rbd</literal>, and &ogw; uses <literal>rgw</literal>.
-   </para>
-  </tip>
-
+<screen>&prompt.cephuser;ceph osd pool create <replaceable>pool_name</replaceable> <replaceable>pg_num</replaceable> <replaceable>pgp_num</replaceable> replicated <replaceable>crush_ruleset_name</replaceable> \
+<replaceable>expected_num_objects</replaceable></screen>
   <para>
-   A pool can be associated with multiple applications, and each application
-   can have its own metadata. You can display the application metadata for a
-   given pool using the following command:
+   To create an erasure coded pool, execute:
   </para>
-
-<screen>&prompt.cephuser;ceph osd pool application get <replaceable>pool_name</replaceable></screen>
- </sect1>
- <sect1 xml:id="ceph-pools-operate">
-  <title>Operating pools</title>
-
+<screen>&prompt.cephuser;ceph osd pool create <replaceable>pool_name</replaceable> <replaceable>pg_num</replaceable> <replaceable>pgp_num</replaceable> erasure <replaceable>erasure_code_profile</replaceable> \
+<replaceable>crush_ruleset_name</replaceable> <replaceable>expected_num_objects</replaceable></screen>
   <para>
-   This section introduces practical information to perform basic tasks with
-   pools. You can find out how to list, create, and delete pools, as well as
-   show pool statistics or manage snapshots of a pool.
+   The <command>ceph osd pool create</command> can fail if you exceed the
+   limit of placement groups per OSD. The limit is set with the option
+   <option>mon_max_pg_per_osd</option>.
   </para>
+  <variablelist>
+   <varlistentry>
+    <term>pool_name</term>
+    <listitem>
+     <para>
+      The name of the pool. It must be unique. This option is required.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>pg_num</term>
+    <listitem>
+     <para>
+      The total number of placement groups for the pool. This option is
+      required. Default value is 8.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>pgp_num</term>
+    <listitem>
+     <para>
+      The total number of placement groups for placement purposes. This should
+      be equal to the total number of placement groups, except for placement
+      group splitting scenarios. This option is required. Default value is 8.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>crush_ruleset_name</term>
+    <listitem>
+     <para>
+      The name of the crush ruleset for this pool. If the specified ruleset
```

Comment:
Since CRUSH is an acronym, and since "crush" is already an English word with a widely accepted meaning quite different from CRUSH, I would suggest to always write CRUSH in ALL-CAPS.